lib/gitsh/tab_completion/dsl.rb

```diff
@@ -0,0 +1,16 @@
+require 'gitsh/tab_completion/dsl/lexer'
+require 'gitsh/tab_completion/dsl/parser'
+
+module Gitsh
+  module TabCompletion
+    module DSL
+      def self.load(path, start_state, env)
+        source = File.read(path)
+        tokens = Lexer.lex(source)
+        factory = Parser.parse(tokens, gitsh_env: env)
+        factory.build(start_state)
+      rescue Errno::ENOENT
+      end
```

Comment:
The silence here makes me suspect that we want a conditional elsewhere instead. I feel like `DSL.load` should complain about a missing file, but that `DSL.load` should only be called if we want such a complaint ... if that makes sense.